system-int-test/src/cmr/system_int_test/utils/subscription_util.clj

```diff
@@ -195,3 +198,28 @@
         result (ingest-subscription subscription options)]
     (index/wait-until-indexed)
     result))
+
+(defn save-umm-granule
```

Comment:
I don't see where this function is used. Delete it.

Reply:
`save-umm-granule` is used in `cmr.system-int-test.ingest.subscription-processing-test` `subscription-job-manual-time-constraint-test` lines 95-100.

It was created to help save custom granules with a specified revision-date.

Reply:
oh, I missed that whole file as it is not expended in the PR diff automatically.